modules/nvidia-gpu-gcp-adding-a-gpu-node.adoc

```diff
@@ -0,0 +1,322 @@
+myclustername-2pt9p-worker-c-6pbg6.c.openshift-qe.internal       Ready      worker                 8h      v1.25.4+77bec7a
+myclustername-2pt9p-worker-gpu-a-wxcr6.c.openshift-qe.internal   Ready      worker                 4h35m   v1.25.4+77bec7a
+----
+
+. View the machines and machine sets that exist in the `openshift-machine-api` namespace by running the following command. Each compute machine set is associated with a different availability zone within the GCP region. The installer automatically load balances compute machines across availability zones.
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                               DESIRED   CURRENT   READY   AVAILABLE   AGE
+myclustername-2pt9p-worker-a       1         1         1       1           8h
+myclustername-2pt9p-worker-b       1         1         1       1           8h
+myclustername-2pt9p-worker-c       1         1                             8h
+myclustername-2pt9p-worker-f       0         0                             8h
+----
+
+. View the machines that exist in the `openshift-machine-api` namespace by running the following command. At this time, there is only one compute machine per machine set, though a compute machine set could be scaled to add a node in a particular region and zone.
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api | grep worker
+----
++
+.Example output
++
+[source,terminal]
+----
+myclustername-2pt9p-worker-a-mxtnz       Running   n2-standard-4   us-central1   us-central1-a   8h
+myclustername-2pt9p-worker-b-9pzzn       Running   n2-standard-4   us-central1   us-central1-b   8h
+myclustername-2pt9p-worker-c-6pbg6       Running   n2-standard-4   us-central1   us-central1-c   8h
+----
+
+. Make a copy of one of the existing compute `MachineSet` definitions and output the result to a JSON file by running the following command. This will be the basis for the GPU-enabled compute machine set definition.
++
+[source,terminal]
+----
+$ oc get machineset myclustername-2pt9p-worker-a -n openshift-machine-api -o json  > <output_file.json>
+----
+
+. Edit the JSON file and make the following changes to the new `MachineSet` definition:
++
+* Rename the machine set `name` by inserting the substring `gpu` in `metadata.name` and in both instances of `machine.openshift.io/cluster-api-machineset`.
+* Change the `machineType` of the new `MachineSet` definition to `a2-highgpu-1g`, which includes an NVIDIA A100 GPU.
++
+[source,terminal]
+----
+jq .spec.template.spec.providerSpec.value.machineType ocp_4.12_machineset-a2-highgpu-1g.json
+
+"a2-highgpu-1g"
+----
++
+The `<output_file.json>` file is saved as `ocp_4.12_machineset-a2-highgpu-1g.json`.
+
+. Update the following fields in `ocp_4.12_machineset-a2-highgpu-1g.json`:
++
+* `.metadata.name` to a name containing `gpu`.
```

Comment:
For the bullets, they aren't complete sentences so they don't need a period after them. I would suggest either making them a sentence by adding a verb to each (for example, "Change the `.metadata.name` to a name containing `gpu`.") or use a "field:action" format (`.metadata.name`: Use a name containing `gpu`) and no period at the end of each.

Reply:
Added the verbs.